doc/tutorials/basic/index.md

```diff
@@ -75,6 +75,18 @@ Now that you've got the basics down, it's time to put your skills to the test:
 - **[Build a Streaming Dashboard](build_streaming_dashboard.md)**
 - **[Build a Chat Bot](build_chatbot.md)**
 
+## Community Tutorials
```

Comment:
We will probably have to update these once in a while.

It would be really awesome if @sophiamyang would update the 3 Ways to Build a Panel Visualization Dashboard blog post. Main points to update would be hvplot.interactive to .rx (code below), button_type+button_style, sizing_mode of plot, the picture showing download numbers.

```python

import panel as pn
import hvplot.pandas

# Load Data
from bokeh.sampledata.autompg import autompg_clean as df

# Make DataFrame Pipeline Interactive
idf = pn.rx(df)

# Define Panel widgets
cylinders = pn.widgets.IntSlider(name='Cylinders', start=4, end=8, step=2)
mfr = pn.widgets.ToggleGroup(
    name='MFR',
    options=['ford', 'chevrolet', 'honda', 'toyota', 'audi'], 
    value=['ford', 'chevrolet', 'honda', 'toyota', 'audi'],
    button_type='primary',
    button_style="outline",
    )
yaxis = pn.widgets.RadioButtonGroup(
    name='Y axis', 
    options=['hp', 'weight'],
    button_type='primary',
    button_style="outline",
)

# Combine pipeline and widgets
rx_pipeline = (
    idf[
        (idf.cyl == cylinders) & 
        (idf.mfr.isin(mfr))
    ]
    .groupby(['origin', 'mpg'])[yaxis].mean()
    .to_frame()
    .reset_index()
    .sort_values(by='mpg')  
    .reset_index(drop=True)
)

# Pipe to hvplot
rx_plot = rx_pipeline.hvplot(x='mpg', y=yaxis, by='origin', color=["#ff6f69", "#ffcc5c", "#88d8b0"], line_width=6, height=400)

# Layout using Template
template = pn.template.FastListTemplate(
    title='Interactive DataFrame Dashboards with hvplot .interactive', 
    sidebar=[cylinders, 'Manufacturers', mfr, 'Y axis' , yaxis],
    main=[pn.pane.HoloViews(rx_plot, sizing_mode="stretch_width")],
    accent="#88d8b0",
)
template.servable()
```

```python

```